### PR TITLE
Add group methods to the custom destination docs in JS, TS and Reason

### PR DESCRIPTION
**Pull request by alex-tpom6oh**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝Changed routes:


| [**…art-custom-destination**](https://docs-80wufacie.avo-dev.app/implementation/start-custom-destination) |
|:-:|
|<a href="https://docs-80wufacie.avo-dev.app/implementation/start-custom-destination">Screenshot of /implementation/start-custom-destination</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/okfDbP-CxCdtJvTFZtheL22NkfIAwQjiqVrV-1n-sZPsFzpFgVxxu8HarwtEi7OJrIEjJQjPRL6XLPiTWt2LDA.png">(view full size)</a> |







Commit 1fb966c16293be0aa0c2ea15d807e19ad1363c18 (https://docs-80wufacie.avo-dev.app).

File: pages/implementation/start-custom-destination.mdx [hunk 1]
```diff
@@ -255,6 +324,34 @@ Call the initAvo function:
...
+  unidentify: () => ();
+  addToGroup: (groupType: string,
+    groupId: string,
+    groupProperties: Js.Json.t
+  ) => ();
+  logEventWithGroup: (
+    eventName: string,
+    eventProperties: Js.Json.t,
+    groupType: string,
+    groupId: string
```
Comment (bragur) [inline pages/implementation/start-custom-destination.mdx:344]: Oooh, I think named arguments would be very nice here

Comment (bragur) [reply pages/implementation/start-custom-destination.mdx:344]:
```suggestion
  logEventWithGroup: (
    ~eventName: string,
    ~eventProperties: Js.Json.t,
    ~groupType: string,
    ~groupId: string
```

Comment (bragur) [reply pages/implementation/start-custom-destination.mdx:344]: or even just for the group arguments

Comment (alex-tpom6oh) [reply pages/implementation/start-custom-destination.mdx:344]: right, for the backward compatibility

Comment (alex-tpom6oh) [reply pages/implementation/start-custom-destination.mdx:344]: ah, it's a new method, so should be fine either way

**Pull request merged**